src/pharmpy/tools/external/nonmem/results.py

```diff
@@ -385,11 +385,11 @@ def _parse_grd(
     last_row = last_row.squeeze(axis=0).rename('gradients')
 
     if (last_row == 0).any():
```

Comment:
Detail perhaps: `final_zero_gradient = (last_row == 0).any()` might be snappier than the if statement